crates/core/tedge_agent/src/main.rs

```diff
@@ -20,17 +20,31 @@ pub struct AgentOpt {
     /// If on also reports DEBUG and TRACE
     #[structopt(long)]
     pub debug: bool,
+
+    /// Start the agent with clean session off, subscribe to the topics, so that no messages are lost
+    #[structopt(short, long)]
+    pub init: bool,
+
+    /// Start the agent with clean session on, drop the old connection and subscriptions
+    #[structopt(short, long)]
+    pub drop: bool,
```

Comment:
My plan was not to add this. But it makes sense at least for testing purpose. However, this requires a strong warning that any pending message will be lost.

```suggestion
    /// Start the agent with clean session on, drop the previous session and subscriptions
    ///
    /// WARNING: All pending messages will be lost.
    #[structopt(short, long)]
    pub drop: bool,
```

Reply:
Yes, renamed.